src/ui/HeaderRenderer.ts

```diff
@@ -535,23 +537,38 @@ export default class HeaderRenderer {
         const sColumns = sShifts.map((d) => d.col);
         that.renderColumns(sColumns, sShifts, d3.select(this), clazz + (clazz.substr(clazz.length - 2) !== '_i' ? '_i' : ''));
       }
-    }).select('div.lu-label').call(dropAble(['application/caleydo-lineup-column-number-ref', 'application/caleydo-lineup-column-number'], (data, d: IMultiLevelColumn, copy) => {
-      const col: Column = resolveDrop(data, copy);
+    };
+
+    $headers.filter((d) => isMultiLevelColumn(d) && (<IMultiLevelColumn>d).canJustAddNumbers).each(renderMultiLevel).select('div.lu-label').call(dropAble(['application/caleydo-lineup-column-number-ref', 'application/caleydo-lineup-column-number'], (data, d: IMultiLevelColumn, copy) => {
```

Comment:
should we store the beginning of the mimetypes as constants, since they're all starting the same (`application/caleydo-lineup`) and then add their suffixes?